api/openapi-spec/swagger.json

```diff
@@ -7525,6 +8513,26 @@
      }
     }
    },
+   "v1.SMBiosConfiguration": {
+    "type": "object",
+    "properties": {
+     "family": {
+      "type": "string"
```

Comment:
We should definitely improve in a follow-up our very minimalistic description of all these fields.